network-store-client/src/main/java/com/powsybl/network/store/client/RestNetworkStoreClient.java

```diff
@@ -848,4 +891,29 @@ public void updateGrounds(UUID networkUuid, List<Resource<GroundAttributes>> gro
             AttributeFilter attributeFilter) {
         updateAll(STR_GROUND, "/networks/{networkUuid}/grounds", groundResources, attributeFilter, networkUuid);
     }
+
+    @Override
+    public Optional<ExtensionAttributes> getExtensionAttributes(UUID networkUuid, int variantNum, ResourceType resourceType, String identifiableId, String extensionName) {
```

Comment:
getIdentifiableExtensionsAttributes

Reply:
find a good naming convention for all methods

Reply:
See discussion https://github.com/powsybl/powsybl-network-store/pull/411#discussion_r1701590040